samples/HelloSignalR/Providers/AuthenticationProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AspNet.Security.OpenIdConnect.Server;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Authentication.OpenIdConnect;
+using AspNet.Security.OpenIdConnect.Extensions;
+
+namespace HelloSignalR.Providers {
+    public class AuthenticationProvider : OpenIdConnectServerProvider {
+        public override Task ValidateClientAuthentication(ValidateClientAuthenticationContext context) {
+            if (context.ClientId == "AspNetContribSample") {
+                context.Validated();
```

Comment:
Hmm... May be because there is no documentation on this and I literally am guessing which one fits best.
It also means that we really need to work on documentation at least part of the time :)